### ISSUE TITLE
about ban master

### ISSUE THREAD

**Issue by Tsin** [DDRace/teeworlds#240]
what was it how did it work why it was removed from ddrace 
could we bring it back ?

because if i understood correctly 
ddmax could setup his own ban master server all our servers would connect to it and check the banned ips am i correct ?

**Comment by heinrich5991** [comment]
Yes, that was the purpose of the banmaster.

**Comment by Tsin** [comment]
yeah aka something nobody knew how to use because nobody wrote any "tut/info" about it

still can we have it back ddmax would rly need such a thingy ^^
